formats/branching.md

```diff
@@ -0,0 +1,59 @@
+# Branching and Pull Requests
+
+This markdown file describes the structure we will follow for branches and how we will tackle the pull requests.
+
+## Branches
+
+Here for this pipeline we have 4 major development items
+
+1. Dataset
+2. Feature extraction
+3. Surrogate modeling
+4. Model identifiability
+
+Each of those major development items will have individual branches. We will be working with branches *under* those major development branches. 
+
+Each branch will have at least one related issue. You can have address more than one issue with a branch but please try to tackle a reasonable number of issues with a single branch. It's okay for you to have/work on more than one branch.
+
+The naming convention for each branch will have the following structure 
+
+```<tag>/<your name>/<number of issue(s) you are addressing>```
```

Comment:
I think writing # issue id will be more useful, thoughts?

Reply:
You are right, it's ambiguous this way :+1: